test/github_pull_requests_test.rb

```diff
@@ -90,6 +90,24 @@ def test_pull_request_status_test_success
     assert receive_test({}, github_slug: "pbrisbin/foo")[:ok], "Expected test of pull request to be true"
   end
 
+  def test_pull_request_status_test_success_and_comment_success
+    @stubs.post("/repos/pbrisbin/foo/statuses/#{"0" * 40}") { |_env| [422, {}, ""] }
```

Comment:
You're gonna hit conflicts rebasing of course. Some tips for the conversion:

* `@stubs` becomes `http_stubs`
* `receive` becomes `service_receive`
* All the `assert` forms should be transformed to their equivalent `expect` form.